firstvoices/backend/models/join_request.py

```diff
@@ -17,18 +17,18 @@ class JoinRequestStatus(models.IntegerChoices):
     APPROVED = 10, _("Approved")  # notify user
 
 
-class JoinRequestReason(models.IntegerChoices):
+class JoinRequestReasonChoices(models.IntegerChoices):
     # enum intentionally has gaps to allow future changes to keep sequential order
-    OTHER = 0, _("Other")
-    LANGUAGE_LEARNER = 10, _("Learning the language")
-    LANGUAGE_TEACHER = 20, _("Teaching the language")
-    FLUENT_SPEAKER = 30, _("Fluent speaker")
-    LANGUAGE_INTEREST = 40, _("Interested in languages")
-    HERITAGE = 50, _("Part of my heritage")
-    COMMUNITY_MEMBER = 60, _("Member of this community/nation")
-    COMMUNITY_STAFF = 70, _("Working with this community/nation")
-    RECONCILIATION = 80, _("Reconciliation")
-    FV_TEAM = 90, _("Part of this FirstVoices Language Team")
+    OTHER = 10, _("Other")
+    LANGUAGE_LEARNER = 20, _("Learning the language")
+    LANGUAGE_TEACHER = 30, _("Teaching the language")
+    FLUENT_SPEAKER = 40, _("Fluent speaker")
+    LANGUAGE_INTEREST = 50, _("Interested in languages")
+    HERITAGE = 60, _("Part of my heritage")
+    COMMUNITY_MEMBER = 70, _("Member of this community/nation")
+    COMMUNITY_STAFF = 80, _("Working with this community/nation")
+    RECONCILIATION = 90, _("Reconciliation")
+    FV_TEAM = 100, _("Part of this FirstVoices Language Team")
```

Comment:
@gmcauliffe Do you need an API to retrieve this list, or are you ok with making a copy of it in the frontend?

Reply:
I'm ok with making a copy in the FE. Thanks :)